@here/harp-mapview/lib/MapView.ts

```diff
@@ -1234,6 +1234,13 @@ export class MapView extends THREE.EventDispatcher {
         return this.m_visibleTileSetOptions.projection;
     }
 
+    set projection(projection: Projection) {
+        this.m_visibleTileSetOptions.projection = projection;
```

Comment:
I think we should maintain proper `geoCenter` by stg like:

```
const x = this.mapView.geoCenter;
// change projection
this.mapView.geoCenter = x;
```

User shouldn't be forced to whole internal known state, if `MapView` knows enough to maintain proper view.

Same applies to camera, with such a basic transformation i'm worried that camera will go crazy when you change from mercator to globe.

Reply:
You're absolutely right. I will fix.